syntax_and_semantics/comments.md

```diff
@@ -1,6 +1,6 @@
 # Comments
```

Comment:
Would you remove this file from the PR? Let's keep this PR focused on the integer specs table.

Reply:
I don't know how that got in there. I didn't make any changes to it since the previous PR.

Reply:
I guess it's because you have both things in master. You should create a branch from crystal-lang/master for each PR to avoid this. The 3 commits about comments should be in their own branch, and the 2 commits relevant to this PR should be in their own branch. 

I don't know if it's possible to modify a PR so that it targets a different branch, so you'll probably have to close this PR and open a new one.